.github/workflows/mac_binary.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+.*'
```

Comment:
It could still work, I'm just not sure if github does anything different since `*` could just be any number of characters afterward with the example I put

Reply:
Sorry was a bit tired yesterday.  Will revert it!